src/components/progress/themes/circular/circular.progress.base.scss

```diff
@@ -27,8 +27,10 @@
 
 [part~='label'],
 [part~='value'] {
-    font-size: calc(var(--diameter) / 3 - var(--stroke-thickness));
-    line-height: normal;
+    font-size: rem(12px);
+    line-height: rem(16px);
+    font-weight: 600;
+    letter-spacing: rem(0.15px);
```

Comment:
So this means we will no longer scale the font size with the size of the progress? Okay.